### PR TITLE
Replace gettext with 'polib' and add test for 'fix_style' function

### LINKED ISSUE #76: Use polib instead of gettext msgcat
[Polib library](https://polib.readthedocs.io) is clossplatform and has a stable API. Can be used to create the differences between pofiles without the need of install `gettext`.

### PR DESCRIPTION
**Pull request by mondeja**
- `gettext` dependency replaced with Python `polib`.
- Added test for `fix_style` function.
- Added type hints for boolean optional arguments.
- Some reformatting to match flake8 and black style.

Closes #76

### CODE REVIEW

File: powrap/powrap.py
Comment (Seluj78) [inline powrap/powrap.py:22]: Should you set explicitly the 79 width in an else clause ?

**Review by Seluj78 — APPROVED** [review]
Thanks for the merge request ! I have not tested your changes yet, and I will wait for @JulienPalard's review before merging

File: powrap/powrap.py
Comment (mondeja) [reply powrap/powrap.py]: Yes, it's safer. But polib declares it as `78`.

**Comment by JulienPalard** [comment]
Hi thanks for contributing, and thanks for your time!

I'd love to see a Python implementation of the msgcat/poedit wrapping sadly there's not.

If we apply this PR, projects using `powrap` will be entierly rewrapped (meaning a colossal diff), while powrap is here to lower the diff in project histories by unifying wrapping convention.

I tried on https://github.com/python/python-docs-fr, a `powrap *.po */*.po`:
```
$ git diff | wc -l
243576
```

So no, polib is not an acceptable replacement.

(Yes we really need to stick to msgcat / poedit wrapping as there's a lot of contributors using poedit, so they don't have to care about powrap at all. On the other hand there's nobody editing files with a polib-compatible wrapping convention).

If you're still interested in dropping `msgcat` dependencies, I started https://github.com/JulienPalard/powrap/tree/from_msgcat that could be a way of doing it.

**Pull request closed without merging**
